.github/workflows/danger.yml

```diff
@@ -1,14 +1,16 @@
-name: iOS Consistency Checks
+name: Run Checks with Danger
 on:
   pull_request:
-    # The restrictions on the types of pull_request event that can trigger this
-    # workflow is intentional. We only want to run these checks when the PR is
-    # opened or reopened, or its branch receives a push.
-    types: [opened, reopened, synchronize]
+    # Because we have a rule that validates the PR labels, we want it to run
+    # when the labels change, not only when a PR is opened/reopened or changes
+    # are pushed to it.
+    types: [opened, reopened, synchronize, labeled, unlabeled]
 
 jobs:
-  check:
+  danger:
     runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Moved the token env definition up here to DRY the code.

One downside is that now other steps can access it as well, but as long as this remains a workflow meant to only run Danger, I don't think we should worry about 3rd party steps misusing the token.

What do you think? Is the DRYness worth it?